client/src/pages/HomePage.js

```diff
@@ -7,7 +7,7 @@ const HomePage = () => {
 	return (
 		<div>
 			<div className="py-24">
-      			<Navbar/>
+					<Navbar/>
```

Comment:
Indentation is unclear

Reply:
I fixed one of the indentation mismatch. I'm not sure why its appearing like this. Did you mix up between tabs and spaces? Anyway, I will look into it.

Reply:
It's ok. You can merge @KillerQueen-BitesZaDusto